package.json

```diff
@@ -37,6 +37,8 @@
   },
   "license": "Apache 2.0",
   "dependencies": {
+    "element-size": "^1.1.1",
+    "lodash": "^3.9.3",
```

Comment:
Ah looks like I use it in the example and didn't add it to the dependancy. Can you move it to the devDependencies since it's not a requirement of the actual component?